### PR TITLE
doc(style): Update docs mobile layout, links for docs and readmes

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
updates docs layout in mobile (text fits inside entire viewport), update links in docs and readme and other informational markdown files, and address css nits.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by mistercrunch** [comment]
Let's merge this prior PR first that goes deeper into the navbar / mobile experience https://github.com/apache/incubator-superset/pull/10853

**Comment by mistercrunch** [comment]
Most of this addressed in #10853 except for the broken link I think.

File: README.md [hunk 1]
```diff
@@ -71,24 +74,43 @@ A modern, enterprise-ready business intelligence web application.
...
-    authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, ...)
-* A lightweight semantic layer, allowing to control how data sources are
-    exposed to the user by defining dimensions and metrics
-* Out of the box support for most SQL-speaking databases
-* Deep integration with Druid allows for Superset to stay blazing fast while
-    slicing and dicing large, realtime datasets
-* Fast loading dashboards with configurable caching
+
+Viewed
+@@ -35,7 +35,7 @@ Superset
```
Comment (rusackas) [inline README.md:79]: Is this code intentional? Looks like github copy/paste stuff

File: README.md [hunk 2]
```diff
@@ -71,24 +74,43 @@ A modern, enterprise-ready business intelligence web application.
...
+  width="500"	  width="500"
+/>	/>
+
+
+A modern, enterprise-ready business intelligence web application.	Apache Superset (Incubating) is a modern, enterprise-ready business intelligence web application. It is fast, lightweight, intuitive, and loaded with options that make it easy for users of all skill sets to explore and visualize their data, from simple pie charts to highly detailed deck.gl geospatial charts.
+
+
+[**Why Superset**](#why-superset) |	[**Why Superset**](#why-superset) |
+[**Supported Databases**](#supported-databases) |	[**Supported Databases**](#supported-databases) |
+@@ -71,25 +71,27 @@ A modern, enterprise-ready business intelligence web application.
```
Comment (rusackas) [inline README.md:89]: More github stuff, if I'm not mistaken

File: docs/src/gatsby-theme-docz/index.tsx [hunk 3]
```diff
@@ -34,6 +34,10 @@ interface Props {
 const docLayout = css`
   display: flex;
   flex-direction: row;
+  .docContainer {
+    word-wrap: break-all;
```
Comment (rusackas) [inline docs/src/gatsby-theme-docz/index.tsx:38]: I think you mean `word-break: break-all;`, but that causes issues, with line-wraps mid-word.

File: docs/src/gatsby-theme-docz/index.tsx [hunk 4]
```diff
@@ -34,6 +34,10 @@ interface Props {
...
+    width: 100%;
```
Comment (rusackas) [inline docs/src/gatsby-theme-docz/index.tsx:39]: Yep, this fixes the problem. I'll undo a flexbox change I was messing with, to the same effect, on another PR.

Note though, that with this div being 100%, and the `.anchorNavStyle` div being `position: fixed`, I'm pretty darn sure that the `docLayout`'s `display: flex` and `flex-direction: row` styles have no effect/purpose.

File: docs/src/components/layout.tsx [hunk 5]
```diff
@@ -46,6 +47,23 @@ const layoutStyles = css`
     border-color: #20a7c9;
     border-radius: 4px;
   }
+  .desktopView {
```
Comment (rusackas) [inline docs/src/components/layout.tsx:50]: are these desktopView/mobileView/tabletView classes used somewhere?

**Review by rusackas — CHANGES_REQUESTED** [review]
Commented on a few things that need addressing. I also made some additional changes that I don't _think_ will conflict, over in #10886 which you might want to take a look at.

File: README.md [hunk 6]
```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
...
+
+A modern, enterprise-ready business intelligence web application.	Apache Superset (Incubating) is a modern, 
+enterprise-ready business intelligence web application. It is fast, lightweight, intuitive, 
+and loaded with options that make it easy for users of all skill sets to explore and visualize 
+their data, from simple pie charts to highly detailed deck.gl geospatial charts.
+
+[**Why Superset**](#why-superset) |	[**Why Superset**](#why-superset) |
+[**Supported Databases**](#supported-databases) |	[**Supported Databases**](#supported-databases) |
+
+## Why Superset	## Why Superset
```
Comment (mistercrunch) [inline README.md:88]: this looks wrong

File: README.md [hunk 7]
```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
...
+
+Superset provides:	Superset provides:
```
Comment (mistercrunch) [inline README.md:90]: odd doubles, wondering what happened here. I highly recommend doing a first sanity pass on your PR when opening the PR on the step prior to "Create PR"

**Review by mistercrunch — CHANGES_REQUESTED** [review]

**Comment by rusackas** [comment]
Looks like there are a couple of linting errors blocking CI, but some merged PRs should fix them. Rebasing this branch should solve it.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -299,31 +299,53 @@ cd incubator-superset
...
+```
+npm run clean
+```
 
 ### Flask server
 
 #### OS Dependencies
 
-Make sure your machine meets the [OS dependencies](https://superset.incubator.apache.org/installation.html#os-dependencies) before following these steps.
+Make sure your machine meets the [OS dependencies](https://superset.incubator.apache.org/docs/installation/installing-from -scratch) before following these steps.
```
Comment (mistercrunch) [inline CONTRIBUTING.md:348]: there's an extra space in the link here... 
but actually the OS dependencies are in another place https://superset.apache.org/docs/installation/installing-superset-from-scratch#os-dependencies

File: README.md [hunk 9]
```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
...
-* A lightweight semantic layer, allowing to control how data sources are
-    exposed to the user by defining dimensions and metrics
-* Out of the box support for most SQL-speaking databases
-* Deep integration with Druid allows for Superset to stay blazing fast while
-    slicing and dicing large, realtime datasets
-* Fast loading dashboards with configurable caching
+
+Viewed
+
+A modern, enterprise-ready business intelligence web application.	Apache Superset (Incubating) is a modern,
```
Comment (mistercrunch) [inline README.md:80]: I think there's a tab `\t` here, let's break the line instead

File: README.md [hunk 10]
```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
...
-    Integration with major
-    authentication backends (database, OpenID, LDAP, OAuth, REMOTE_USER, ...)
-* A lightweight semantic layer, allowing to control how data sources are
-    exposed to the user by defining dimensions and metrics
-* Out of the box support for most SQL-speaking databases
-* Deep integration with Druid allows for Superset to stay blazing fast while
-    slicing and dicing large, realtime datasets
-* Fast loading dashboards with configurable caching
+
+Viewed
```
Comment (mistercrunch) [inline README.md:78]: why does it say "Viewed" here?

File: README.md [hunk 11]
```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
...
+
+A modern, enterprise-ready business intelligence web application.	Apache Superset (Incubating) is a modern,
+enterprise-ready business intelligence web application. It is fast, lightweight, intuitive,
+and loaded with options that make it easy for users of all skill sets to explore and visualize
+their data, from simple pie charts to highly detailed deck.gl geospatial charts.
```
Comment (mistercrunch) [inline README.md:83]: actually i think this whole paragraph is duplicated from above

File: README.md [hunk 12]
```diff
@@ -71,24 +74,45 @@ A modern, enterprise-ready business intelligence web application.
...
+- Enterprise-ready authentication with integration with major authentication providers
+  (database, OpenID, LDAP, OAuth & REMOTE_USER through Flask AppBuilder)
+- An extensible, high-granularity security/permission model allowing intricate rules on who can access
+  individual features and the dataset
+- A simple semantic layer, allowing users to control how data sources are displayed in the UI by defining
+  which fields should show up in which drop-down and which aggregation and function metrics are made available to the user
+- Integration with most SQL-speaking RDBMS through SQLAlchemy
+- Deep integration with Druid.io
+
+Superset is cloud-native and designed to be highly available. It was designed to scale out to large,
```
Comment (mistercrunch) [inline README.md:103]: wondering where these README updates are coming from, is that content that Kevin copied/wrote/recommended?

**Pull request closed without merging**